src/commands/utils.rs

```diff
@@ -223,6 +238,29 @@ pub fn finalized_emer_txs(revaultd: &RevaultD) -> Result<Vec<BitcoinTransaction>
         .map_err(|e| e.into())
 }
 
+/// Get the Unvault transaction for a given vault
+pub fn unvault_tx(
```

Comment:
I think, in the future we should move these helpers in a `scripts` or `consensus` module with a struct handling the descriptors and the secp_ctx, encapsulating the derivation etc. Revaultd would store this struct and we use it like this: 
`revaultd.consensus.unvault_tx(&vault.derivation_index)` or `revaultd.generator.unvault_tx(&derivation_index)`
(I dont know yet which name)